### ISSUE TITLE
Why mages suck, why it's a symptom of a larger problem and how to fix (this is code related)

### ISSUE THREAD

**Issue by dmhenderson76** [DarkCastleMUD/DarkCastle#37]
So after spending a bunch of time building a new zone and looking for meaningful skill boosts for all classes - it has become clear that some classes simply do not benefit from skill bump increases and it hampers the imms' ability to work meaningful content into zones for them.  Let me give you some examples and then I will suggest a solution:

Mages (no one plays them):
* Most of their damage unsurprisingly comes from spells.  Issue is, these spells have FLAT damage amounts that are not dictated by skill for some reason (with the exception of fireball).  Increasing a mage's hellstream or solar gate skill does absolutely nothing to ramp their damage.  Mages are stuck in a box where as soon as they gain a spell, it is what it is.  Mages don't have the ability to "specialize" at all - even minus professions.  If spell damage were tied to the mage's skill level in that spell - and frankly, I have no idea why it isn't already - mages could potentially specialize in certain kinds of magic. 
-Mage specializes in fire damage so he boosts fireshield and hellstream
-Mage specializes in energy so bolt/directed bolt are bootsed and deal more damage
-Mage specializes in non-elemental damage so they look for gear that boots meteor swarm

...which brings me to Barbs.  They have the same problem.  Berserk is already crazy powerful and probably doesn't need to be amped damage-wise but there could be perks if skill levels are high enough.  I went through the entire barb skill repertoire and you know how many skills they have that make a difference when boosted (outside of outright failure)?  The answer is one (dodge).  There are literally no other skills a barb can boost that change the way their character plays.  Don't think that the only answer is outright damage either - maybe for mages in terms of elemental damage, but not barbs.  How about
* Berserk skill over <x%> allows barb to "calm down" after a 3 round lag and disengage
* Rage skill bumps damage slightly but also increases penalties slightly.
* Blood fury heals a little more if skill is boosted etc

If we want to diversify races and classes on DC - the structure is already there BUT these skill percentages have to mean something.  Other class examples:
* Cleric/pal sanc damage reduction should be dependent upon skill level - Heal already does this!
* Holy aura duration should depend on skill level
* Consecrate/Desecrate should have a skill % factored in instead of dealing damage off pure alignment OR have skill affect spell duration.
* Maybe make creeping death ramp bingo % on skill level instead of raw daamage.  Possibilities are limitless!

tl;dr:  
* Not enough classes on DC benefit from skill % increases and several classes are just locked out entirely.  This needs to change.
* Skills that are skill% dependent do not lean heavily enough upon skill %s to matter.  Ex:  +20 behead translates to roughly a +1% behead chance increase to a paladin.  Imms need revised guidance on what +skill%s to put on items.
*  IMO skills should cap lower but increases should matter more and have a noticeable impact upon gameplay.

**Comment by dmhenderson76** [comment]
Add to this, it's entirely reasonable to make it so that certain race/char combinations cannot reach skill thresholds naturally.  

Ex:  In order to headbutt a huge mob requires a headbutt skill over 80 which a warrior normally cannot hit, but with specialized gear maybe they can?  There needs to be a better stratification of player abilities where people can pick and choose what they're good at.  We need more of the headbutt approach.

Builders can help to give players options - but it won't matter if 75%+ of available skills/spells do not benefit from skilL% increase.

**Comment by bdunavant** [comment]
@jhhudso I am planning on closing this, as it's discussion on what should be done, and this is not a discussion board.   Do we have a place for discussion/debate that is more appropriate for me to direct the traffic there?   Only bugs, or confirmed features that have already been debated/approved w/blessing should have issue tickets.

**Comment by jhhudso** [comment]
Originally it was http://forums.dcastle.org which is still up and works.
